pages/test_analytics/flaky_test_assignment.md

```diff
@@ -0,0 +1,32 @@
+# Flaky test assignment
+
+Customers on the [Pro and Enterprise plans](https://buildkite.com/pricing) can assign flaky tests to [Teams](/docs/team_management/permissions).
```

Comment:
Whoops, I forgot one more fix...

```suggestion
Customers on the [Pro and Enterprise plans](https://buildkite.com/pricing) can assign flaky tests to [Teams](/docs/team-management/permissions).
```

While our docs system resolves underscores to hyphens, it's probably better to use the real deal here.